src/openforms/appointments/tests/test_views.py

```diff
@@ -20,13 +20,14 @@
     SubmissionFactory,
     SubmissionStepFactory,
 )
+from openforms.tests.mixins import FrontendRedirectMixin
 
 from ..tokens import submission_appointment_token_generator
 from .factories import AppointmentFactory, AppointmentInfoFactory
 
 
 @freeze_time("2021-07-15T21:15:00Z")
-class VerifyCancelAppointmentLinkViewTests(TestCase):
+class VerifyCancelAppointmentLinkViewTests(TestCase, FrontendRedirectMixin):
```

Comment:
```suggestion
class VerifyCancelAppointmentLinkViewTests(FrontendRedirectMixin, TestCase):
```
mixins come before the base class for consistent MRO